File: web3rpc/rpc-specs/code-samples/python/eth/block/getUncleByBlockHashAndIndex.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0xb8deae63002d2b6aa33247c8ef545383ee0fd2282ac9b49dbbb74114389ddb5c"
uncleIndex = "0x1"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_uncle_by_block_hash_and_index(blockHash, uncleIndex)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getUncleByBlockHashAndIndex.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockHash = '0xc9dbfbab67e9a0508bcb3f95ae408023668cef431b805592781a821926715b8a'
    const uncleIndex = '0x1'

    sdk.eth.getUncleByBlockHashAndIndex(blockHash, uncleIndex, {}, (err, data, response) => {
        console.log(data);
    });
}
)()